services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/document/EnhancedDocument.java

```diff
@@ -440,13 +449,12 @@ interface Builder {
         Builder addJson(String attributeName, String json);
 
         /**
-         * Convenience builder methods that sets an attribute of this document for the specified key attribute name and value.
-         *
-         * @param keyAttrName  Name of the attribute that needs to be added in the Document.
-         * @param keyAttrValue The value that needs to be set.
+         * Appends an attribute of name attributeName with specified value of the give EnhancedDocument.
```

Comment:
Does this method add a nested document to the builder?

Reply:
Yes